documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -2,7 +2,12 @@
 
 ## Version 10.0.0
 
-[company_name] [solution_name] has been renamed [detect_product_long]. As such, page links and documentation URLs have been updated accordingly. Update any [solution_name] documentation bookmarks you may have.
+[company_name] [solution_name] has been renamed [detect_product_long] with page links, documentation, and other URLs updated accordingly. Update any [detect_product_short] documentation, or other bookmarks you may have. See the [Domain Change FAQ](https://community.blackduck.com/s/article/Black-Duck-Domain-Change-FAQ).
+* As part of this activity, sig-repo.synopsys.com and detect.synopsys.com are being deprecated. Please make use of repo.blackduck.com and detect.blackduck.com respectively. 
+	* After February 2025, [detect_product_short] 10.0.0 script download details will only be available via detect.blackduck.com.
```

Comment:
The same for Detect 9 and 8 scripts, right?

Reply:
Updated.